src/plugins/curator/components/teaser-picker-field/Option.jsx

```diff
@@ -0,0 +1,59 @@
+import { components } from 'react-select';
+import { Media } from '@triniti/admin-ui-plugin/components';
+import classNames from 'classnames';
+import damUrl from '@triniti/cms/plugins/dam/utils/damUrl';
+import NodeStatus from '@gdbots/schemas/gdbots/ncr/enums/NodeStatus';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const statusColorMap = Object.values(NodeStatus).reduce((acc, cur) => {
```

Comment:
@albertgcsula this map seems unnecessary since you can just render the value of the enum as a string or use the status variable itself.

Reply:
@gdbrown okay, removed.